engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/repository-details-manage-ostree-branches.controller.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * @ngdoc object
+ * @name  Bastion.repositories.controller:RepositoryManageContentController
+ *
+ * @requires $scope
+ * @requires translate
+ * @requires Repository
+ *
+ * @description
+ *   Provides the functionality for the repository details pane.
+ */
+angular.module('Bastion.repositories').controller('RepositoryManageOstreeBranchesController',
+    ['$scope', 'translate', 'Repository',
+    function ($scope, translate, Repository) {
+
```

Comment:
Are all the functions contained within being tested or accessed in the template? If not, better to turn them into private functions so they aren't exposed in an 'API' like fashion where I would expect them to be tested and usable. i.e. using `function nameOfFunction() {}` syntax.